io.github.swanux.hbud.yml

```
@@ -0,0 +1,139 @@
+app-id: io.github.swanux.hbud
+runtime: org.gnome.Platform
+runtime-version: '41'
+sdk: org.gnome.Sdk
+command: io.github.swanux.hbud
+
+finish-args:
+  - --share=network
+  - --socket=wayland
+  - --socket=x11
```

```suggestion
  - --socket=fallback-x11
```

if it actually support wayland?

---

Right... It seems that the GTK integration of libVLC video playback (through Gtk.DrawingArea) doesn't support Wayland. - There's an experimental OpenGL method in the unstable 4.0 branch of libVLC, but I'd wait for it to become upstream before implementing. 

So for now, I should omit wayland.